package.json

```diff
@@ -2,8 +2,7 @@
   "name": "graasp-plugin-search",
   "version": "0.1.0",
   "description": "",
-  "main": "dist/index.js",
```

Comment:
Actually could you keep the index as the main entry file? And keep types as well.

Reply:
I restore the index. Currently this repo does not need extra type definnitions, so I omit the types file